test/library/packages/Shlex/Shlex.chpl

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright 2020-2022 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+  A lexical anayzer class for simple shell-like syntaxes.
+  Implementation and design inspired by Python's Shlex module: 
+  <https://docs.python.org/3/library/shlex.html>
+  This module provides support for split, join and quote operations.
+*/
+
+module Shlex {
+  private use List;
+  private use Regex;
+  class shlex{
```

Comment:
```suggestion
  class shlex {
```

Reply:
Note: I _think_ committing these suggestions will generate an error in the DCO signed author checks, so I believe it's better for you to make the changes yourself in the code.